### PR TITLE
docs: update logs integration and calculation types content 

### PR DESCRIPTION
**Pull request by jasuade**
**What is this feature?**

Update of a couple of documents for last version:
- Logs Integration Explore: Remove `Toggle field visibility` as there is no toggle anymore
- New Calculation Types:  Add latest Calculations to the table 

**Why do we need this feature?**

**Who is this feature for?**
Docs

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by imatwawana** [comment]
@jasuade - Thanks for the contribution! I just want to confirm which version you mean, 10.0 or 9.5?
Also, can you please make sure to sign the CLA (link in [this comment](https://github.com/grafana/grafana/pull/68074#issuecomment-1539947021)).

**Review by imatwawana — COMMENTED** [review]
Thanks for the update! I've left some wording editing suggestions. Once the version question is sorted out, then we can get this labeled properly and merged.

File: docs/sources/explore/logs-integration.md [hunk 1]
```diff
@@ -9,25 +9,21 @@ weight: 15
...
 
 - [Elasticsearch]({{< relref "../datasources/elasticsearch/" >}})
 - [InfluxDB]({{< relref "../datasources/influxdb/" >}})
 - [Loki]({{< relref "../datasources/loki/" >}})
 
-During an infrastructure monitoring and incident response, you can dig deeper into the metrics and logs to find the cause. Explore also allows you to correlate metrics and logs by viewing them side-by-side. This creates a new debugging workflow:
-
-1. Receive an alert.
-1. Drill down and examine metrics.
-1. Drill down again and search logs related to the metric and time interval (and in the future, distributed traces).
```
[the remark this answers is not in the document]
Comment (jasuade) [reply docs/sources/explore/logs-integration.md:22]: To be honest, this is a high level list of steps on "what to do" when an incident is triggered, but you may want to use log Explorer for other purposes. 
In general, I do know think it provides extra value, just more text.
Also,` in the future, distributed traces` well, we already have Traces and log to traces functionality up and running so... seemed outdated.

**Comment by jasuade** [comment]
Hey Isabel! 
I did it actually for version 9.5, but it may apply to older versions, although not sure if we update documentation retrospectively.

Also, I signed the CLA!

**Comment by imatwawana** [comment]
> Hey Isabel! I did it actually for version 9.5, but it may apply to older versions, although not sure if we update documentation retrospectively.
> 
> Also, I signed the CLA!

Thanks for getting that signed! 

We do actually update retroactively to 9.3 currently. Is it painful for you to check if these changes apply back that far? If so, let me know and I can look into it.

**Comment by jasuade** [comment]
Hey @imatwawana I could not find the exact change log, but I am pretty confident the changes that are included (Toggle field visibility) has not been in Logs Explore since before v9, so we can put this retroactively.

File: docs/sources/explore/logs-integration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/explore/logs-integration.md:22]: Sounds good!

**Review by imatwawana — APPROVED** [review]

**Pull request merged**